src/Headings/HeadingOne.tsx

```diff
@@ -0,0 +1,34 @@
+import React, {
+  FunctionComponent,
+  ReactElement,
+  useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { BaseTheme } from '../Theme';
+
+const Heading = styled.h1`
+  font-family: ${({ theme }): string => theme.font.heading.family};
+  font-size: ${({ theme }): string => theme.font.heading.size};
+  font-weight: ${({ theme }): string => theme.font.heading.weight};
+  color: ${({ theme }): string => theme.font.heading.color};
```

Comment:
These should use the `theme.font.title` attributes, since the `h1` element should only be used for the top page title.

Reply:
Fixed in 40f0bd7